### DISCUSSION TITLE
pressure overlay

### DISCUSSION THREAD

**Discussion by Aripaule** [aaronwmorris/indi-allsky#1717] [Q&A] [answered]
hello, 

About the overlay, I would like to add pressure, but I didn't find a variable for that, any ideas on how to do it?

my custom overlay:
Dew point {sensor_user_2:0.0f}°{temp_unit:s}
#color:255,255.0
test. {sensor_user_18}
# color:127,0,255
D.H. Status {dew_heater_status}
#color:255,255.0
fan status {fan_status}
#color:255,255.0
Temperature {sensor_user_10:0.0f}°{temp_unit:s}
#color:50,50,150
sky temp {sensor_user_15}
#color:50,50,150

![image](https://github.com/user-attachments/assets/b88368ee-e990-47d0-a41c-9491e4e57f60)

![image](https://github.com/user-attachments/assets/21712ffa-e2c1-41a8-a3c1-dcb96b6f16e5)

ty

**Comment by aaronwmorris** [comment]
What sensor do you have for detecting pressure?  You can just post your Support Info bundle which will have the answer.

**Reply by Aripaule** [reply]
bme 280

**Reply by aaronwmorris** [reply, marked as answer]
If you have the BME280 configured starting at slot 10, then the pressure data will be in the 3rd offset from 10, so `{sensor_user_12:0.1f}`

**Reply by Aripaule** [reply]
OK. I understood the logic of the list.

ty
